components/image.css

```diff
@@ -0,0 +1,11 @@
+.display-image{
+    margin-top: 1rem;
+    display: flex;
+    justify-content: space-between;
+    max-width: 30rem;
+}
+
+.img-rounded{
+    border-radius: 100%;
+}
```

Comment:
50% also gives same affect any number above 50 works, but usually we use 50%

Reply:
Noted, thanks.